step1-02/demo/README.md

```diff
@@ -84,3 +84,5 @@ Open up the browser inspector and target our "all" button. You'll notice that th
 > **Specificity** states that regardless of cascade, the selector with the highest specificity wins
 
 To fix this problem we need to either reduce the specificity of our button styles, or increase the specificity of the selected style. In this situation we will add `.filter` in front of the `.selected` selector, because the selected style only applies to the filter anyway.
+## Next Step
```

Comment:
Can you add a blank line before the heading in each file to ensure it gets formatted properly?

Reply:
Can you explain me exactly what should i do?